src/logic/dictionaryUtils.ts

```diff
@@ -40,12 +63,13 @@ export function matchesToCommanderHelper(
     playerNameFilter?: string,
     startDate?: Date,
 ): { [id: string]: Commander } {
-    const filteredMatches = filterMatchesByDate(matches, startDate);
+    const filteredMatches = filterMatchesBySeatCount(filterMatchesByDate(matches, startDate), 4);
```

Comment:
based on what i'm seeing here, we may need to update Player.ts to include "validMatchCount" as a new property. This will be different from matches.count. This is a bigger change, so I'm ok doing that in a follow up PR if you want (there will be a little bit of time where winrate is being miscalculated because it is taking the lower wincount divided by ALL matchCount (when we only want matchCount with 4 players)

Reply:
We'll then tag the 3 player matches as not counting toward stats calculations-- it's simply there for posterity.

Reply:
I think 3 player games can still count toward player commander color pie graph though.